src/main/java/uk/gov/gchq/palisade/service/request/DataRequestConfig.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Crown Copyright
+ * Copyright 2020 Crown Copyright
```

Comment:
not a required change since you haven't significantly changed this file or in this case changed it at all

Reply:
Yeah, I think i touched it in some way and then reverted any changes.